bin/update-env

```diff
@@ -0,0 +1,50 @@
+#! /usr/bin/env bash
+# shellcheck source-path=..
+
+set -euo pipefail
+
+#### Detect Toolkit Project Root ####
+# if realpath is not available, create a semi-equivalent function
+command -v realpath >/dev/null 2>&1 || realpath() {
+  [[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"
+}
+SCRIPT_PATH="$(realpath "${BASH_SOURCE[0]}")"
+SCRIPT_DIR="$(dirname "$SCRIPT_PATH")"
+TOOLKIT_ROOT="$(realpath "$SCRIPT_DIR/..")"
+if [[ ! -d "$TOOLKIT_ROOT/bin" ]] || [[ ! -d "$TOOLKIT_ROOT/config" ]]; then
+  echo "ERROR: could not find root of overleaf-toolkit project (inferred project root as '$TOOLKIT_ROOT')"
+  exit 1
+fi
+
+source "$TOOLKIT_ROOT/lib/shared-functions.sh"
+
+function usage() {
+  echo "Usage: bin/update-env"
+  echo ""
+  echo "Updates config/variables.env, adding new environment variables"
+  echo ""
+  echo "This program will pass any extra flags to docker compose,"
+  echo "for example: 'bin/up -d' will run in detached mode"
+}
+
+function __main__() {
+  if [[ "${1:-null}" == "help" ]] || [[ "${1:-null}" == "--help" ]]; then
+    usage
+    exit
+  fi
+
+  # read_image_version
+  echo "This script will update your config/variables.env."
+  echo "We recommend backing up your config with bin/backup-config."
+  prompt "Do you want to continue?"
+
+  echo "Creating backup file config/__old-variables.env"
+  cp config/variables.env config/__old-variables.env
```

Comment:
If I run this twice, my earlier backup would be gone. WDYT about adding a timestamp into the path?

Reply:
Done in https://github.com/overleaf/toolkit/pull/214/commits/8b9fc04f55ff5d93a9c0e0732a9a1200a091e967